src/main/scala/com/microsoft/hyperspace/index/sources/delta/DeltaLakeRelation.scala

```diff
@@ -194,9 +200,13 @@ class DeltaLakeRelation(spark: SparkSession, override val plan: LogicalRelation)
         .get
         .asInstanceOf[IndexLogEntry]
     }
-    // TODO: Currently assume all versions of index data exist.
-    //  Need to check and remove candidate indexes.
-    //  See https://github.com/microsoft/hyperspace/issues/387
+
+    // Get available index log versions.
+    val availableLogVersions =
```

Comment:
`activeLogVersions`? and I think you can remove the comment unless you want to explain "why" you are getting active logs.